sources/packages/api/src/route-controllers/institution-locations/institution-location.aest.controller.ts

```diff
@@ -36,4 +43,35 @@ export class InstitutionLocationAESTController extends BaseController {
       institutionId,
     );
   }
+
+  /**
+   * Controller method to retrieve institution location by id.
+   * @param institutionId
+   * @param locationId
+   * @returns institution location.
+   */
+  @Get(":locationId/institution/:institutionId")
```

Comment:
Apparently, there is no reason to have the `institutionId` here, please remove it.